### ISSUE TITLE
Deep Classification, Embedding & Text Generation (E2) - Liu et al et al 2020

### ISSUE THREAD

**Issue by HyunkuKwon** [UChicago-CCA-2021/Readings-Responses#35]
Liu, Meijun, Yi Bu, Chongyan Chen, Jian Xu, Diafeng Li, Yan Leng, Richard Freeman...Ying Ding. 2020. “[Can Pandemics Transform Scientific Novelty? Evidence from COVID-19.](https://arxiv.org/pdf/2009.12500.pdf)” arXiv:2009.12500.

**Comment by jacyanthis** [comment]
It looks like the causal inference here is difference-in-differences (DiD) between countries where the first reported coronavirus case has been reported versus not been reported. Do you find this convincing, and what do you think of recent critiques of staggered DiD designs ([example](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3794018))? Is this a good causal inference approach for text analysis?

**Comment by romanticmonkey** [comment]
The paper show that novelty and parachute collabs goes up and international collabs go down during COVID-19, which makes me wonder how would these three variables perform among war-state countries (e.g. scientific breakthroughs during WWI, WWII). Since pandemics and wars are both "life-and-death" situations, would they show similar trends in scientific novelty? (Of course the international collabs will occur among allies.) 

In addition, I wonder if scientific funding would have great effect on novelty. I'm not sure what motivated Newton during the Great Plague, but for contemporary researchers, might monetary resources be a significant motivation for innovations? (e.g. with more funds, scientists become more imaginative and dare try out expensive, complicated methods.)

**Comment by Rui-echo-Pan** [comment]
My question is about the measure of the "novelty". They use the measure of BioBERT to capture the distance between two bio-entities in each entity pair extracted from CORD-19 papers. And it seems that both the technique and the research findings are robust because the paper finds a similar pattern in the 2003 SARS pandemic when the average novelty score boomed up at the specific time. 

But my question is, do you think such measure meaningful? I mean, when a new virus occurs and a new pandemic occurs, the research seems will naturally focus on the newly coming problem/phenomenon. So the boosting of novelty seems natural and not surprising. (But I do think the relation between collaboration and novelty is interesting to me!!!)

As a response to @jacyanthis about the DiD techniques, I do think DID is useful and appropriate here to identify the treatment effect, and may not be too related to text analysis?

**Comment by MOTOKU666** [comment]
In terms of how disasters influence collaboration and innovation, I feel like the Covid 19 might not be a good example if we didn't consider the effect of the global political context on them.

In addition, I think it would be more interesting to compare examples such as SARS, MERS(with more details), H1N1 pandemic, and focus more on the lagged effect of the corresponding pandemic. It's possible that such disasters enable more investment in scientific research and infrastructures. The long-term effect may worth exploring.

**Comment by zshibing1** [comment]
It is not clear how they address the reverse causality in this case - has scientific novelty affected the pandemics in the process to some extent as well?

**Comment by hesongrun** [comment]
What is the size of the corpus the authors use to detect novelty? How does it compare to traditional ones?

**Comment by ming-cui** [comment]
I'm also concerned with the operationalization of novelty. Intuitively, I would take a look at which journal the paper was published in. Generally speaking, papers published in top journals, such as Science and Nature, are innovative. Would you think that the impact of journals should be integrated into the calculation of novelty score?

**Comment by xxicheng** [comment]
Agreed with @MOTOKU666 , I am also more interested in the long-term effects brought by the pandemic. It seems quite natural that new researches focus on covid 19 since it is such a big global outbreak.

**Comment by william-wei-zhu** [comment]
will the effect of covid-19 on international collaboration be long-lasting? or will the patterns of collaboration resume to pre-covid times when the world goes back to normal?

**Comment by mingtao-gao** [comment]
I don't find the results surprising as we are going through a global pandemic. My concern is also the operationalization of novelty in this case, as many have mentioned. Since one of two major data sources is the COVID-19 Open Research
Dataset, which covers 58,728 research articles about COVID-19 and related historical coronaviruses. Will the result naturally tend to show low novelty since those papers are targeting on one topic?

**Comment by RobertoBarrosoLuque** [comment]
The authors find that "international collaborations are more novel than their counterparts", "coronavirus research has become more novel since the outbreak", etc. I am wondering if there is similar literature applying word embedding models or BERT to understand novelty in other domains?

**Comment by Bin-ary-Li** [comment]
I always have a distrust about the actual contribution brought by the COVID-19 research boom. Some of them feel like they were being rushed out or done for the sake of doing things related to COVID. Why should we trust an exploratory analysis like this one is not one of them?

**Comment by toecn** [comment]
What are the mechanisms that explain an increase in "parachuting  collaboration" during a global health crisis?
